package/runc/0001-Separate-Device-handling-for-default-AllowDevices.patch

```diff
@@ -262,3 +249,20 @@ index 8c7fb774..965ec4ff 100644
  			}
  			found = true
  		}
+diff --git a/vendor/github.com/opencontainers/cgroups/devices/config/device.go b/vendor/github.com/opencontainers/cgroups/devices/config/device.go
+index 05ad3ef8..e6b8a732 100644
+--- a/vendor/github.com/opencontainers/cgroups/devices/config/device.go
++++ b/vendor/github.com/opencontainers/cgroups/devices/config/device.go
+@@ -24,6 +24,9 @@ type Device struct {
+ 
+ 	// Gid of the device.
+ 	Gid uint32 `json:"gid"`
++
++	// Is Default Device
++	IsDefault bool `json:"is_default"`
+ }
+ 
+ // Permissions is a cgroupv1-style string to represent device access. It
+-- 
+2.50.0
```

Comment:
Nit, but couldn't we settle on using `git format-patch --no-signature` for patches that we keep in Git, to avoid diff noise when updating patches?

Reply:
I also use `--no-numbered` usually, that is what Buildroot also advice to do.